src/cljs/org/broadinstitute/firecloud_ui/page/method_repo.cljs

```diff
@@ -6,39 +6,33 @@
    [org.broadinstitute.firecloud-ui.common.table :as table]
    [org.broadinstitute.firecloud-ui.endpoints :as endpoints]
    [org.broadinstitute.firecloud-ui.page.workspace.method-config-importer :as importmc]
+   [org.broadinstitute.firecloud-ui.utils :as utils]
    ))
 
 
 (react/defc MethodsList
   {:render
    (fn [{:keys [props state]}]
-     (if (:show-import-overlay? @state)
-       [comps/Dialog
-        {:blocking? true
-         :width "80%"
-         :dismiss-self #(swap! state dissoc :show-import-overlay?)
-         :content  (importmc/render-import-overlay
-                     nil ; nil workspace because entrypoint is  from method repo
-                     #(swap! state dissoc :show-import-overlay?) ; on-close
-                     #(swap! state dissoc :show-import-overlay?) ; on-import because entry from method repo
-                     (:selected-method @state))}]
      [table/Table
-      {:empty-message "No methods to display."
-       :columns [{:header "Namespace" :starting-width 150 :sort-by :value}
-                 {:header "Name" :starting-width 150 :sort-by :value
-                  :content-renderer (fn [row-index msm]
-                                      [:a {:style {:color (:button-blue style/colors)
-                                                   :textDecoration "none"}
-                                           :href "javascript:;"
-                                           :onClick #(swap! state assoc :show-import-overlay? true
-                                                      :selected-method (str msm))}
-                                       (str msm)])}
+      {:empty-message "No method configurations to display."
+       :columns [{:header "Method" :starting-width 250 :sort-by :value}
+                 {:header "Method Synopsis" :starting-width 150 :sort-by :value }
+                 {:header "Configuration Name" :starting-width 150 :sort-by :value}
+                 {:header "Namespace" :starting-width 150 :sort-by :value}
+                 {:header "Snapshot ID" :starting-width 150 :sort-by :value  }
                  {:header "Synopsis" :starting-width 500 :sort-by :value}]
        :data (map (fn [m]
-                    [(m "namespace")
-                     (m "name")
-                     (m "synopsis")])
-               (:methods props))}]))})
+                    (let [method-obj (m "method")
+                          m-name (method-obj"name")
+                          m-ns (method-obj "namespace")
+                          m-s-id (method-obj "snapshotId")]
+                      [(clojure.string/join ":" [m-name m-ns m-s-id])
+                       (method-obj "synopsis")
+                       (m "name")
```

Comment:
The comment line I added is underneath the line I'm describing ... that's a general standard. So, in this case, I'm referring to line 31: "(m "name")" That column was linked, and I think it should still be as that would bring up the config import. I think ... the production UI is empty for me so I can't do a real side-by-side test.